boards/intel/ish/board.cmake

```diff
@@ -1,4 +1,13 @@
 # SPDX-License-Identifier: Apache-2.0
 
+set(SUPPORTED_EMU_PLATFORMS simics)
```

Comment:
you set `SUPPORTED_EMU_PLATFORMS` here, but if the board is a `CONFIG_BOARD_INTEL_ISH_5_8_0` then you load `common/simics.board.cmake` which also sets (overwrites) the `SUPPORTED_EMU_PLATFORMS` with the exact same value, is that intentional ?

https://github.com/zephyrproject-rtos/zephyr/blob/998f72cf3d3ec5214b21617078e46afc31ef6060/boards/intel/ish/board.cmake#L9

Another question, are all intel ish boards supporting `simics` or only `intel ish 5.8.0` ?

Reply:
> Another question, are all intel ish boards supporting simics or only intel ish 5.8.0 ?

all can run in simics, this one is the one I have a setup for, so right now it is the only one working this way.

Reply:
> you set `SUPPORTED_EMU_PLATFORMS` here, but if the board is a `CONFIG_BOARD_INTEL_ISH_5_8_0` then you load `common/simics.board.cmake` which also sets (overwrites) the `SUPPORTED_EMU_PLATFORMS` with the exact same value, is that intentional ?

fixed